---
{"_attributes":{"version":"4.4.4+","urlbase":"https://bugs.ecmascript.org/","maintainer":"dherman@mozilla.com"},"bug":{"bug_id":4521,"creation_ts":"2015-08-21 14:15:00 -0700","short_desc":"25.4.5.3.1 PerformPromiseThen: Use assertion in step 9","delta_ts":"2015-10-26 12:43:42 -0700","product":"ECMA-262 Edition 6","component":"technical issues","version":"unspecified","rep_platform":"All","op_sys":"All","bug_status":"RESOLVED","resolution":"FIXED","priority":"Normal","bug_severity":"enhancement","everconfirmed":true,"reporter":{"uid":"andrebargull","name":"André Bargull"},"assigned_to":{"uid":"allen","name":"Allen Wirfs-Brock"},"cc":"brterlso","long_desc":[{"commentid":14682,"comment_count":0,"who":{"uid":"andrebargull","name":"André Bargull"},"bug_when":"2015-08-21 14:15:07 -0700","thetext":"25.4.5.3.1 PerformPromiseThen ( promise, onFulfilled, onRejected, resultCapability )\n\n\nChange step 9 to use:\n---\nAssert: The value of promise's [[PromiseState]] internal slot is \"rejected\".\n---"},{"commentid":14874,"comment_count":1,"who":{"uid":"brterlso","name":"Brian Terlson"},"bug_when":"2015-10-26 12:43:29 -0700","thetext":"Fixed in ES2016 Draft (26532a9)."}]}}
---
